src/Store/StoreThread.re

```diff
@@ -86,7 +86,7 @@ let start =
       ~minimize,
       ~window: option(Revery.Window.t),
       ~filesToOpen=[],
-      ~overriddenExtensionsDir=?,
+      ~overriddenExtensionsDir=None,
```

Comment:
Maybe I'm being an idiot...but I thought that in this context `?` and `None` are equivalent?

However when doing `esy` I get that `cliOptions.overriddenExtensionsDir` is an `option(string)` and it is expecting a `string`....which I don't really understand. The docs here https://reasonml.github.io/docs/en/function#optional-labeled-arguments seem to suggest that it should work.

Reply:
Ah, ya, `None` vs `?` is very confusing... looks good 👍